src/components/forms/SubmitTemplate/server-action.ts

```diff
@@ -36,6 +41,8 @@ export async function submitTemplate(formData: FormData) {
       templateEntity.fields!.content
     );
 
+    await deleteSession(sessionId);
```

Comment:
what are your thoughts on the ordering here? deleteSession at the end like this has the risk of being able to submit templates twice - but deleteSession at the start might mean that they don't get submitted at all

Reply:
Hmm... It should probably be before the send email TBH.

If the user happens to go back and forward in the process and 2 templates get created seems like a real edge case. The only thing I could think of to prevent this happening is to tie a template onto the session via the sessionId.

Reply:
Changed order to delete session _before_ sending an email.